target/linux/ramips/dts/rt5350_olimex_rt5350f-olinuxino-evb-plus.dts

```diff
@@ -0,0 +1,119 @@
+/dts-v1/;
+
+#include "rt5350_olimex_rt5350f-olinuxino-evb.dts"
+
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "olimex,rt5350f-olinuxino-evb-plus", "ralink,rt5350-soc";
+	model = "Olimex RT5350F-OLinuXino-EVB+";
+
+	aliases {
+		spi0 = &spi_mmc;
+		spi1 = &spi_uext;
+	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		acin {
+			gpio-export,name = "acin";
+			gpios = <&gpio0 21 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	spi_mmc: spi-mmc {
+		compatible = "spi-gpio";
+		#address-cells = <1>;
+		ranges;
+
+		gpio-sck = <&gpio0 4 GPIO_ACTIVE_HIGH>;
+		gpio-miso = <&gpio0 6 GPIO_ACTIVE_HIGH>;
+		gpio-mosi = <&gpio0 5 GPIO_ACTIVE_HIGH>;
+		cs-gpios = <&gpio0 3 GPIO_ACTIVE_HIGH>, <&gpio1 5 GPIO_ACTIVE_HIGH>;
+		num-chipselects = <2>;
+
+		m25p80@0 {
+			compatible = "jedec,spi-nor";
+			reg = <0>;
+			spi-max-frequency = <10000000>;
+
+			partitions {
+				compatible = "fixed-partitions";
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partition@0 {
+					label = "u-boot";
+					reg = <0x0 0x30000>;
+					read-only;
+				};
+
+				partition@30000 {
+					label = "u-boot-env";
+					reg = <0x30000 0x10000>;
+					read-only;
+				};
+
+				factory: partition@40000 {
+					label = "factory";
+					reg = <0x40000 0x10000>;
+					read-only;
+				};
+
+				partition@50000 {
+					compatible = "denx,uimage";
+					label = "firmware";
+					reg = <0x50000 0x7b0000>;
+				};
+			};
+		};
+
+		mmc-slot@0 {
+			compatible = "mmc-spi-slot";
+			reg = <1>;
+
+			voltage-ranges = <3200 3400>;
+			spi-max-frequency = <1000000>;
+
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	spi_uext: spi-uext {
+		compatible = "spi-gpio";
+		#address-cells = <1>;
+		ranges;
+
+		gpio-sck = <&gpio0 20 GPIO_ACTIVE_HIGH>;
+		gpio-miso = <&gpio0 18 GPIO_ACTIVE_HIGH>;
+		gpio-mosi = <&gpio0 17 GPIO_ACTIVE_HIGH>;
+		cs-gpios = <&gpio0 19 GPIO_ACTIVE_HIGH>;
+		num-chipselects = <1>;
+
+		spidev@1 {
+			compatible = "spidev";
+			status = "okay";
+			reg = <0>;
+			spi-max-frequency = <1000000>;
+		};
+	};
+};
+
+&spi0 {
+	status = "disabled";
+	/delete-node/ m25p80@0;
```

Comment:
This should be resolved by the shared DTSi, i.e. only including this node in the other DTS, but not in the DTSI.